testsuite/cluster-test/src/health/mod.rs

```diff
@@ -74,12 +76,12 @@ pub trait HealthCheck {
 
 pub struct HealthCheckRunner {
     cluster: Cluster,
-    health_checks: Vec<Box<dyn HealthCheck>>,
+    health_checks: Vec<Box<dyn HealthCheck + Send>>,
```

Comment:
Oh, one note

You can just make `HealthCheck: Send` in declaration of the HealthCheck, rather then adding `+ Send` everywhere

Reply:
ok, I will update in the next PR

Reply:
You can update it here and use `r=andll` after to merge it